src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -116,14 +111,55 @@ export default class TableComponent extends ComponentBase<azdata.TableComponentP
 		}
 	}
 
-	public static transformData(rows: string[][], columns: any[]): { [key: string]: string }[] {
+	private static createIconColumn(col: azdata.TableColumn): Slick.Column<any> {
```

Comment:
Make these functions generic `createIconColumn<T extends Slick.SlickData>` please. It doesn't matter too much in this specific case but it's just better code design to not use any if we can avoid it.